pkg/deploy-manager/config.go

```diff
@@ -27,23 +27,31 @@ const DefaultStorageClusterName = "test-storagecluster"
 // DefaultStorageClassRBD is the name of the ceph rbd storage class the test suite installs
 const DefaultStorageClassRBD = DefaultStorageClusterName + "-ceph-rbd"
 
-// MinOSDsCount represents the minimum number of OSDs required for this testsuite to run.
-const MinOSDsCount = 3
+// DefaultMinOSDsCount represents the minimum number of OSDs required for this testsuite to run.
+const DefaultMinOSDsCount = 3
 
 func init() {
 	ocsv1.SchemeBuilder.AddToScheme(scheme.Scheme)
 	rookcephv1.SchemeBuilder.AddToScheme(scheme.Scheme)
 	nbv1.SchemeBuilder.AddToScheme(scheme.Scheme)
 }
 
+type StorageClusterOptions struct {
+}
+
+type storageClusterConfig struct {
+	minOSDsCount int
+}
```

Comment:
indeed, i wanted to ask the same and was distracted reading through the code :wink:

Reply:
Thanks for making me think harder. I have dropped StorageClusterOptions. To configure, we will have methods on the DeployManager, instead of having a big struct of options.